react/features/dialog/components/Dialog.js

```diff
@@ -0,0 +1,154 @@
+/* global APP */
+
+import ModalDialog from '@atlaskit/modal-dialog';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { SpeakerStats } from '../../speaker-stats';
+import { hideDialog, showDialog } from '../actions';
+
+const dialogs = {
```

Comment:
I like what you did here, good work!  FTR, on a next iteration I'd like to see some `DialogRegistry` entity so all dialogs can register themselves there once we port them.